src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2309,26 +2495,70 @@ subroutine PlantCrop(p, leafcn_in, jday, &
+         idop              =>    cnveg_state_inst%idop_patch                     , & ! Output: [integer  (:) ]  date of planting
+         iyop              =>    cnveg_state_inst%iyop_patch                     , & ! Output: [integer  (:) ]  year of planting
          leafc_xfer        =>    cnveg_carbonstate_inst%leafc_xfer_patch         , & ! Output: [real(r8) (:) ]  (gC/m2)   leaf C transfer
          leafn_xfer        =>    cnveg_nitrogenstate_inst%leafn_xfer_patch       , & ! Output: [real(r8) (:) ]  (gN/m2)   leaf N transfer
          crop_seedc_to_leaf =>   cnveg_carbonflux_inst%crop_seedc_to_leaf_patch  , & ! Output: [real(r8) (:) ]  (gC/m2/s) seed source to leaf
-         crop_seedn_to_leaf =>   cnveg_nitrogenflux_inst%crop_seedn_to_leaf_patch & ! Output: [real(r8) (:) ]  (gN/m2/s) seed source to leaf
+         crop_seedn_to_leaf =>   cnveg_nitrogenflux_inst%crop_seedn_to_leaf_patch, & ! Output: [real(r8) (:) ]  (gN/m2/s) seed source to leaf
+         hybgdd            =>    pftcon%hybgdd                                 , & ! Input:  [real(r8) (:) ]
+         gddmin            =>    pftcon%gddmin                                 , & ! Input:
+         gdd020            =>    temperature_inst%gdd020_patch                 , & ! Input:  [real(r8) (:) ]  20 yr mean of gdd0
+         gdd820            =>    temperature_inst%gdd820_patch                 , & ! Input:  [real(r8) (:) ]  20 yr mean of gdd8
+         gdd1020           =>    temperature_inst%gdd1020_patch                  & ! Input:  [real(r8) (:) ]  20 yr mean of gdd10
          )
 
       ! impose limit on growing season length needed
       ! for crop maturity - for cold weather constraints
       croplive(p)  = .true.
       idop(p)      = jday
+      iyop(p)      = kyr
       harvdate(p)  = NOT_Harvested
-      sowing_count(p) = sowing_count(p) + 1
-      crop_inst%sdates_thisyr(p,sowing_count(p)) = jday
+      s = sowing_count(p) + 1
+
+      sowing_count(p) = s
+      if (s < mxsowings) then
+         next_rx_sdate(p) = crop_inst%rx_sdates_thisyr(p, s+1)
+      else
+         next_rx_sdate(p) = -1
+      endif
+      crop_inst%sdates_thisyr(p,s) = real(jday, r8)
+
+      this_sowing_reason = 0._r8
+      if (do_plant_prescribed) then
+          this_sowing_reason = 10._r8
+      end if
+      if (do_plant_normal) then
+          this_sowing_reason = this_sowing_reason + 1._r8
+      else if (do_plant_lastchance) then
+          this_sowing_reason = this_sowing_reason + 2._r8
+      end if
+      sowing_reason(p,s) = this_sowing_reason
+
+      this_sowing_reason = 0._r8
+      if (do_plant_normal) then
+          this_sowing_reason = this_sowing_reason + 1._r8
+      else if (do_plant_lastchance) then
+          this_sowing_reason = this_sowing_reason + 2._r8
+      end if
+      sowing_reason(p,sowing_count(p)) = this_sowing_reason
+      crop_inst%sowing_reason_patch(p) = this_sowing_reason
```

Comment:
- [ ] There are two nearly-duplicate blocks here with some subtle differences. Is this intentional? If so, can you please add an explanatory comment saying what's going on in each of these blocks, or how/why they differ?

Reply:
Nope, completely unintentional. Not sure how it happened! Resolved with a463b724f.